MIP 9 Community Greenlight Poll - FEI (Fei USD) - May 16, 2022.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Community Greenlight Poll - FEI (Fei USD) - May 16, 2022
+summary: Signal your support or opposition to prioritising onboarding FEI (Fei USD).
+discussion_link: https://forum.makerdao.com/t/fei-mip6-collateral-onboarding-application/14895
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Greenlight
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
+start_date: 2002-05-16T16:00:00
+end_date: 2002-05-30T16:00:00
+---
+# Poll: Community Greenlight Poll - FEI (Fei USD) - May 16, 2022
+
+The Governance Facilitators have placed a governance poll into the [voting system](https://vote.makerdao.com/polling) as part of their responsibilities as defined in [MIP9: Community Greenlight](https://mips.makerdao.com/mips/details/MIP9). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, May 16, at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll asks whether the asset FEI (Fei USD) should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state).
+
+Please review the collateral onboarding application [thread](https://forum.makerdao.com/t/fei-mip6-collateral-onboarding-application/14895) to inform your position before voting.
+
+## Outcomes
+
+**Regardless of the winning option the following actions will be taken at the conclusion of this poll:**
+* The 'greenlight score' will be calculated for FEI (Fei USD) using the formula `Score = 'Yes' Votes - 'No' Votes`
+* The 'greenlight score' for FEI (Fei USD) will be added to the [Collateral Prioritisation Sheet](https://docs.google.com/spreadsheets/d/1IX9e2fyfz7djtDMKn5gMyGsyFxHoY75GncMbAjnSXrM/edit#gid=0) by the Collateral Engineering Services Core Unit.
```

Comment:
This link is outdated - not your fault, the template is wrong. New link: https://docs.google.com/spreadsheets/d/1PDf_CzhGa7mLuOUfX6Bz3WrnCjDRhIjmu-vDZMMw4Qc/edit#gid=1077340672

It's also now called the Collateral Status Index.

I'll update the template